downloader/src/main/kotlin/VersionControlSystem.kt

```diff
@@ -73,7 +73,10 @@ abstract class VersionControlSystem {
                     try {
                         it.isValid()
                     } catch (e: IOException) {
-                        log.debug { "Exception while trying VCS ${it.getType()}, treating it as non-applicable." }
+                        log.debug {
+                            "Exception while validating ${it.getType()} working tree, treating it as non-applicable."
```

Comment:
Maybe also add the message? `...non-applicable: ${e.message}`

Reply:
Done.